portfolio.adobe.com.stage.json

```diff
@@ -0,0 +1,40 @@
+{
+  "providerId":"portfolio.adobe.com",
+  "providerName":"Adobe Portfolio",
+  "serviceId":"stage",
+  "serviceName":"Adobe Portfolio Stage Websites",
+  "version":1,
+  "description":"Enables a domain to work with Adobe Portfolio",
+  "records":[
+    {
+      "type":"CNAME",
+      "host": "www",
+      "pointsTo":"@",
+      "ttl":3600
```

Comment:
Maybe 600 seconds / 10 minutes for Stage. It will make testing changes easier to verify, especially if you're testing with the same couple of domain names over and over.

Reply:
Updated.